app/views/organisations/_show_organisation.html.erb

```diff
@@ -49,16 +49,34 @@
   <% end %>
 </div>
 
+<% if @organisation.ordered_featured_documents.length %>
+  <%= render "govuk_publishing_components/components/heading", {
```

Comment:
This will also need to go in a grid column-two-thirds.

Reply:
Fixed.